### PR TITLE
fix UnicodeEncodeError in analytics

### PR DESCRIPTION
**Pull request by nickpell**
https://sentry.io/dimagi/commcarehq/issues/657457176/

### CODE REVIEW

**Review by calellowitz — APPROVED** [review]

**Review by esoergel — APPROVED** [review]

**Comment by orangejenny** [comment]
@calellowitz / @nickpell can you summarize the impact of this? Looks like any kiss events over the past ~week that use non-ascii chars would have failed...which is likely just the "other persona" field since that's (I think) the only place where we send user input as part of an event...so likely just this single event. Yes?

**Comment by nickpell** [comment]
@orangejenny It looks very likely that this was a single event:
https://sentry.io/dimagi/commcarehq/?environment=__all_environments__&query=UnicodeEncodeError

Good to get the properties into the expected format though, so avoid issues when switching to py3.

Note: I did mess this up the first time around, which caused kiss errors for about ~30 minutes until I did a hotfix deploy (see fix [here](https://github.com/dimagi/commcare-hq/pull/21628))

**Comment by orangejenny** [comment]
@nickpell Cool, thanks for the explanation.
